docker/docker-entrypoint.sh

```diff
@@ -4,14 +4,22 @@
 TEMPFILE="$(mktemp)"
 
 # Set values in inventory-management-system-settings.json from environment variables
-sed -e "s|\"imsApiUrl\": \".*\"|\"imsApiUrl\": \"$IMS_API_URL\"|" \
-    -e "s|\"osApiUrl\": \".*\"|\"osApiUrl\": \"$OS_API_URL\"|" \
-    -e "s|\"maxAttachmentSizeBytes\": \".*\"|\"maxAttachmentSizeBytes\": \"$MAX_ATTACHMENT_SIZE_BYTES\"|" \
-    -e "s|\"attachmentAllowedFileExtensions\": \".*\"|\"attachmentAllowedFileExtensions\": \"$ATTACHMENT_ALLOWED_FILE_EXTENSIONS\"|" \
-    -e "s|\"imageAllowedFileExtensions\": \".*\"|\"imageAllowedFileExtensions\": \"$IMAGE_ALLOWED_FILE_EXTENSIONS\"|" \
-    -e "s|\"maxImageSizeBytes\": \".*\"|\"maxImageSizeBytes\": \"$MAX_IMAGE_SIZE_BYTES\"|" \
-    -e "s|\"pluginHost\": \".*\"|\"pluginHost\": \"$PLUGIN_HOST\"|" \
-    /usr/local/apache2/htdocs/inventory-management-system-settings.json > "$TEMPFILE"
+jq \
+  --arg imsApiUrl "$IMS_API_URL" \
+  --arg osApiUrl "$OS_API_URL" \
+  --argjson maxAttachmentSizeBytes $MAX_ATTACHMENT_SIZE_BYTES \
+  --argjson attachmentAllowedFileExtensions "$ATTACHMENT_ALLOWED_FILE_EXTENSIONS" \
+  --argjson imageAllowedFileExtensions "$IMAGE_ALLOWED_FILE_EXTENSIONS" \
+  --argjson maxImageSizeBytes $MAX_IMAGE_SIZE_BYTES \
+  --arg pluginHost "$PLUGIN_HOST" \
+  '.imsApiUrl = $imsApiUrl |
+   .osApiUrl = $osApiUrl |
+   .maxAttachmentSizeBytes = $maxAttachmentSizeBytes |
+   .attachmentAllowedFileExtensions = $attachmentAllowedFileExtensions |
+   .imageAllowedFileExtensions = $imageAllowedFileExtensions |
+   .maxImageSizeBytes = $maxImageSizeBytes |
+   .pluginHost = $pluginHost' \
+  public/inventory-management-system-settings.example.json > config.json
```

Comment:
```suggestion
  public/inventory-management-system-settings.example.json > "$TEMPFILE"
```

Reply:
the path was also incorrect so i changed that in the lastest commit